index.js

```diff
@@ -66,6 +68,26 @@ app.delete('/api/v1/favorites/:id', (request, response) => {
     });
 });
 
+app.get('/api/v1/playlists', (request, response) => {
+  database('playlists')
+  .innerJoin('playlists_favorites', 'playlists.id', 'playlists_favorites.playlist_id')
+  .innerJoin('favorites', 'favorites.id', 'playlists_favorites.favorite_id')
+  .select(['playlists.id', 'playlists.title', database.raw("JSON_AGG(favorites) as favorites")])
+  .groupBy('playlists.id', 'playlists_favorites.id', 'favorites.id')
+  .then(playlists => {
```

Comment:
Very well done working out this advanced query.

Reply:
Thanks to Maddie for the help!